### DISCUSSION TITLE
Homepage by genre

### DISCUSSION THREAD

**Discussion by bharathganji** [jellyfin/jellyfin#10262] [General]
was wondering if there was a way to make the home page to show media" by genres 

It would look something like this:

Action-------------

Comedy-----------

Drama-----------

Science Fiction-------------

**Comment by blastilonic** [comment]
I completely agree — this would be an incredibly valuable feature.

For me, this is one of the most important improvements still missing. Having a homepage with a large rotating banner, followed by customizable rows such as:
- Genres 
- Recently added
- Most popular / most watched
- Continue watching
- And similar sections for movies, TV shows, music, etc.

That would make the whole experience feel much more modern and intuitive.

There is the Home Screen Sections plugin for Jellyfin, but unfortunately it doesn’t work properly on TV apps, which is where this feature would make the biggest difference.

Please consider adding this officially — it would greatly enhance the user experience across all devices.
